examples/event/lv_example_event_4.py

```diff
@@ -0,0 +1,52 @@
+class LV_Example_Event_4:
+
+    def __init__(self):
+        #
+        # Demonstrate the usage of draw event
+        #
+        self.size = 0
+        self.size_dec = False
+        self.cont = lv.obj(lv.scr_act())
+        self.cont.set_size(200, 200)
+        self.cont.center()
+        self.cont.add_event_cb(self.event_cb, lv.EVENT.DRAW_PART_END, None)
+        lv.timer_create(self.timer_cb, 30, None)
+        
+    def timer_cb(self,timer) :
+        self.cont.invalidate()
+        if self.size_dec :
+            self.size -= 1
+        else :
+            self.size += 1
+
+        if self.size == 50 :
+            self.size_dec = True
+        elif self.size == 0:
+            self.size_dec = False
+
+    def event_cb(self,e) :
+        obj = e.get_target()
+        dsc = lv.obj_draw_part_dsc_t.__cast__(e.get_draw_part_dsc())
```

Comment:
Is the type cast really needed here?
I think that `get_draw_part_dsc()` already returns a struct of type `lv_obj_draw_part_dsc_t`.